app/views/user/show.html.erb

```diff
@@ -1,2 +1,183 @@
 
+<div class="container">
+<center>
+ <br>
+ <br>
+<h1 >Your Profile, <%=current_user.user_datum.full_name%></h1>
+</center>
 
+
+ <br>
+ <br>
+
+
+
+<div class="card-deck">
+  <div class="card">
+    <img src="<%=current_user.own_photos.first.image_one%>" class="card-img-top" alt="...">
+    <div class="card-body">
+      <h5 class="card-title">Photo 1</h5>
+      <p class="card-text"></p>
+    </div>
+    <div class="card-footer">
+    <form action="/create_user" method="post" class="form-horizontal" enctype="multipart/form-data">
+              <input type="file" id="avatar" name="avatar" class="form-control">
+
+    </form>
+    </div>
+  </div>
+  <div class="card">
+    <img src="<%=current_user.own_photos.first.image_three%>" class="card-img-top" alt="...">
+    <div class="card-body">
+      <h5 class="card-title">Photo 2</h5>
+      <p class="card-text"></p>
+    </div>
+    <div class="card-footer">
+    <form action="/create_user" method="post" class="form-horizontal" enctype="multipart/form-data">
+              <input type="file" id="avatar" name="avatar" class="form-control">
+    </form>
+    </div>
+  </div>
+  <div class="card">
+    <img src="<%=current_user.own_photos.first.image_three%>" class="card-img-top" alt="...">
+    <div class="card-body">
+      <h5 class="card-title">Photo 3</h5>
+      <p class="card-text"></p>
+    </div>
+    <div class="card-footer">
+    <form action="/create_user" method="post" class="form-horizontal" enctype="multipart/form-data">
+              <input type="file" id="avatar" name="avatar" class="form-control">
+
+    </form>
+    </div>
+  </div>
+</div>
+
+
+
+ <br>
+ <br>
+
+<form action="/updateprofile" method: "get">
+  <div class="form-group">
+    <label for="name"> Name </label>
+    <input type="text" class="form-control" id="name" value="<%=current_user.user_datum.full_name%>" name="query_name">
+  </div>
+
+<% m_array = [18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36] %>
+  <label for="political">Age</label>
+  <select class="form-control" id="age" placeholder="yes" name="query_age">
+    <% m_array.each do |number|%>
+      <option <%= "selected" if number == current_user.user_datum.age%>>
+        <%= number %>
+      </option>
+      <%end%>
+    </select>
```

Comment:
There's [a nice guide](https://guides.rubyonrails.org/form_helpers.html#making-select-boxes-with-ease) on using the `select` form helper in a form. It includes an option for choosing the selected value, which mean no more if statement!